docs/DeveloperGuide.md

```diff
@@ -591,9 +591,21 @@ testers are expected to do more *exploratory* testing.
    1. Type `addf` in the command box and enter.
     <br>**Expected:** A form modal appears, containing empty text input fields for name, duration, portion, ingredients, steps, and tags, as well as a `Cancel` and `Save` button on the bottom right.
```

Comment:
```md
<kbd>Cancel</kbd>
```

Reply:
Will add these keyboard shortcut formatting after current changes to UG are merged.